server/command/service.go

```diff
@@ -265,29 +269,60 @@ func (s *service) ExecuteCommand(pluginContext *plugin.Context, commandArgs *mod
 		pluginContext: pluginContext,
 		commandArgs:   commandArgs,
 	}
+	loc := s.i18n.GetUserLocalizer(params.commandArgs.UserId)
 	if pluginContext == nil || commandArgs == nil {
-		return errorOut(params, errors.New("invalid arguments to command.Handler. Please contact your system administrator"))
+		return s.errorOut(params, errors.New(s.i18n.LocalizeDefaultMessage(loc, &nsi18n.Message{
+			ID:    "apps.command.error.invalidArguments",
+			Other: "invalid arguments to command.Handler. Please contact your system administrator",
+		})))
 	}
 
 	conf := s.conf.GetMattermostConfig().Config()
 	enableOAuthServiceProvider := conf.ServiceSettings.EnableOAuthServiceProvider
 	if enableOAuthServiceProvider == nil || !*enableOAuthServiceProvider {
-		return errorOut(params, errors.Errorf("the system setting `Enable OAuth 2.0 Service Provider` needs to be enabled in order for the Apps plugin to work. Please go to %s/admin_console/integrations/integration_management and enable it.", commandArgs.SiteURL))
+		return s.errorOut(params, errors.New(s.i18n.LocalizeWithConfig(loc, &nsi18n.LocalizeConfig{
+			DefaultMessage: &nsi18n.Message{
+				ID:    "apps.command.enableOAuth",
+				Other: "the system setting `Enable OAuth 2.0 Service Provider` needs to be enabled in order for the Apps plugin to work. Please go to {{.URL}} and enable it.",
```

Comment:
Can we capitalize this sentence and the next one pertaining to bot accounts?

Reply:
We shouldn't. They are passed to error out, which will form the sentence as `Command failed. Error: **{{.Error}}**`.